Frontend/src/components/Table/Row.svelte

```diff
@@ -12,68 +13,26 @@
       return col.display ? col.display(item[col.key]) : item[col.key];
     }
   };
+
+  let cellBackgroundColors = new Array(columns.length).fill("white");
+  $: cellBackgroundColorsFunction(item).then(
+    (newCellBackgroundColors) => (cellBackgroundColors = newCellBackgroundColors)
```

Comment:
I guess this allows for async loading of background color?

Reply:
Yes exactly. It basically does the same as the {await} in the HTML part but requires less duplicated code in this case.